src/main/java/uk/gov/hmcts/reform/hmc/repository/PartyRelationshipDetailsRepository.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.hmcts.reform.hmc.repository;
+
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.Propagation;
+import org.springframework.transaction.annotation.Transactional;
+import uk.gov.hmcts.reform.hmc.data.PartyRelationshipDetailsEntity;
+
+import java.util.List;
+
+@Transactional(propagation = Propagation.REQUIRED)
+@Repository
+public interface PartyRelationshipDetailsRepository extends CrudRepository<PartyRelationshipDetailsEntity, Long> {
```

Comment:
Not sure if this is needed now - think it was for testing.  Will check and remove if not needed

Reply:
Removed in https://github.com/hmcts/hmc-cft-hearing-service/pull/153/commits/e430840b5052c4fef7fcddd4e3f82f9670491048